src/modules/markCompleted.js

```diff
@@ -0,0 +1,24 @@
+import { TasksClass } from './tasksClass.js';
+
+// mark task complete
+export const markComplete = () => {
+  //  get the elements
+  const taskBoxEls = document.querySelectorAll('.task-select-input');
+  const taskStatus = new TasksClass();
+  let taskBoxValue = false;
+  // get index and checkbox value by iterating
+  taskBoxEls.forEach((theEl, theElIndex) => {
+    // theEl.checked = true;
```

Comment:
As part of the [ruby best practices](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md) kindly remove this commented code from your codebase. Leaving commented code in your codebase is generally not a good practice. Commented-out code can clutter your codebase, make it harder to read, and can potentially lead to confusion or outdated information. If you need to review a previous version of your code, you can always use git history. :pray:

Reply:
I appreciate the reminder.